src/frontend/src/lib/types/token.ts

```diff
@@ -25,3 +25,5 @@ export type RequiredToken = Required<Token>;
 export type OptionToken = Token | undefined | null;
 export type OptionTokenId = TokenId | undefined | null;
 export type OptionTokenStandard = TokenStandard | undefined | null;
+
+export type DisplayToken = Token & { show?: boolean };
```

Comment:
Not a strong opinion but, I would maybe declare it in a separate module. That way `Token` remains kind of scoped. Maybe a module `manage-token.ts`.

Reply:
on it